controller/src/test/java/io/pravega/controller/eventprocessor/EventProcessorGroupTest.java

```diff
@@ -139,15 +139,12 @@ public void testFailingSealReaderGroupInShutdown() throws CheckpointStoreExcepti
     }
 
     @Test(timeout = 10000)
-    public void testFailingCellShutdown() throws CheckpointStoreException {
+    public void testShutdown() throws CheckpointStoreException {
         this.requestEventProcessors = system.createEventProcessorGroup(requestConfig, checkpointStore, rebalanceExecutor);
         requestEventProcessors.awaitRunning();
         assertTrue(requestEventProcessors.isRunning());
-
         Position mockReaderPosition = mock(Position.class);
         doReturn(ImmutableMap.of("reader1", mockReaderPosition)).when(checkpointStore).sealReaderGroup("host1", "scaleGroup");
-        String exceptionMsg = "Exception marking reader offline.";
-        doThrow(new RuntimeException(exceptionMsg)).when(reader).closeAt(any());
```

Comment:
Do we need a seperate test where `reader.closeAt(..)` throws an exception ?

Reply:
No we don't because even if reader.closeAt() throws an exception that does impact cell.awaitTerminated() and the code flow in the shutdown method.